src/customFileParser/unpackGeometry/InstancedMesh.ts

```diff
@@ -0,0 +1,89 @@
+import * as THREE from 'three';
+import { InstancedMeshGroup, InstancedMesh, InstancedMeshCollection } from '../../geometry/InstancedMeshGroup';
+import { CompressedGeometryData, UncompressedValues } from './../sharedFileParserTypes';
+import PropertyLoader from './../PropertyLoader';
+import { xAxis, yAxis, zAxis } from './../../constants';
+import SceneStats from './../../SceneStats';
+import Sector from './../../Sector';
+
+const matrix = new THREE.Matrix4();
+const rotation = new THREE.Matrix4();
+
+export default function unpackInstancedMeshes(
+  rootSector: Sector,
+  uncompressedValues: UncompressedValues,
+  sectorPathToInstancedMeshData: {[path: string]: CompressedGeometryData},
+  sceneStats: SceneStats,
+  treeIndexNodeIdMap: number[],
```

Comment:
Yeah having treeIndexNodeIdMap as `number[]` instead of a map makes sense I think.